.gitattributes

```diff
@@ -3,17 +3,18 @@
 
 # Denote text files explicitly, normalize line endings to LF on checkin and forbid conversion to CRLF on checkout
 # Without eol=lf, core.autocrlf might come in effect
-*.html  eol=lf diff=html
-*.css   eol=lf
-*.js    eol=lf
-*.sql   eol=lf
-*.php   eol=lf diff=php
-*.yml   eol=lf
-*.xml   eol=lf
-*.dist  eol=lf
-*.conf  eol=lf
-*.json  eol=lf
-*.lock  eol=lf
+*.html      eol=lf diff=html
+*.css       eol=lf
+*.js        eol=lf
+*.sql       eol=lf
+*.php       eol=lf diff=php
+*.yml       eol=lf
+*.xml       eol=lf
+*.dist      eol=lf
+*.conf      eol=lf
+*.json      eol=lf
+*.lock      eol=lf
+/codecept   eol=lf
```

Comment:
Please revert changes made to .gitattributes and Dockerfile.
If you want to improve them, please raise a new pull request.